<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Canvas</class>
 <widget class="QMainWindow" name="Canvas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>736</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
      <item>
       <widget class="QListWidget" name="scenesList">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>800</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="maximumSize">
           <size>
            <width>250</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Settings</string>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>20</y>
             <width>213</width>
             <height>173</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="settingsLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetMinimumSize</enum>
            </property>
            <item>
             <layout class="QVBoxLayout" name="projectionPlaneLayout">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Projection Plane Distance</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QSpinBox" name="projectionPlaneSpinBox">
                  <property name="maximum">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="projectionPlaneSlider">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="maximum">
                   <number>4</number>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="cameraPositionLayout">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Camera Position</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QDoubleSpinBox" name="xCameraPos">
                  <property name="minimum">
                   <double>-10</double>
                  </property>
                  <property name="maximum">
                   <double>10</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="yCameraPos">
                  <property name="minimum">
                   <double>-10</double>
                  </property>
                  <property name="maximum">
                   <double>10</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="zCameraPos">
                  <property name="minimum">
                   <double>-10</double>
                  </property>
                  <property name="maximum">
                   <double>10</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="canvasSizeLayout">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Canvas Size</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QDoubleSpinBox" name="canvasWidth">
                  <property name="minimum">
                   <double>-10</double>
                  </property>
                  <property name="maximum">
                   <double>10</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDoubleSpinBox" name="canvasHeight">
                  <property name="minimum">
                   <double>-10</double>
                  </property>
                  <property name="maximum">
                   <double>10</double>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="tValuesLayout">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>T Values</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>T Min</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="tMin">
                    <property name="minimum">
                     <double>1</double>
                    </property>
                    <property name="maximum">
                     <double>10</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>T Max</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QDoubleSpinBox" name="tMax">
                    <property name="minimum">
                     <double>1</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>736</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
